AdobeStockAsset/etc/di.xml

```diff
@@ -10,5 +10,8 @@
     <preference for="Magento\AdobeStockAssetApi\Api\Data\AssetInterface" type="Magento\AdobeStockAsset\Model\Asset"/>
     <preference for="Magento\AdobeStockAssetApi\Api\Data\KeywordInterface" type="Magento\AdobeStockAsset\Model\Keyword"/>
     <preference for="Magento\AdobeStockAssetApi\Api\Data\AssetSearchResultsInterface" type="Magento\AdobeStockAsset\Model\SearchResults" />
+    <type name="Magento\Config\Model\Config\Backend\Encrypted">
```

Comment:
Implemented plugin will be called for every field in Magento that has `Magento\Config\Model\Config\Backend\Encrypted` backend model. I'm not sure that we want to run suck check on all encrypted fields. It's incorrect behavior.

Reply:
Hello @sidolov ,

Yes, you are right. I'm so stupid to use plugin here. I will update this code.